### DISCUSSION TITLE
Two-way control not working with ReactJS (NextJS)

### DISCUSSION THREAD

**Discussion by Skythz** [nolimits4web/swiper#6535] [Q&A]
Tried copy and pasting the example code in the docs:
```
import React, { useState } from 'react';
import { Controller } from 'swiper';
import { Swiper, SwiperSlide } from 'swiper/react';

const App = () => {
  // store swiper instances
  const [firstSwiper, setFirstSwiper] = useState(null);
  const [secondSwiper, setSecondSwiper] = useState(null);

  return (
    <main>
      <Swiper
        modules={[Controller]}
        onSwiper={setFirstSwiper}
        controller={{ control: secondSwiper }}
      >
        {/* ... */}
      </Swiper>

      <Swiper
        modules={[Controller]}
        onSwiper={setSecondSwiper}
        controller={{ control: firstSwiper }}
      >
        {/* ... */}
      </Swiper>
    </main>
  );
};
```

And got the following error: 
"TypeError: Cannot read properties of undefined (reading 'length')"
![image](https://user-images.githubusercontent.com/44848783/228635589-b5bdbdc2-b48e-4312-8f2e-77cb0a1eab77.png)

It doesn't have anything to do with the rest of my code as I've experimented with it using a new empty file on the same project.
The lines causing the error are:
`controller={{ control: firstSwiper }}`
and 
`controller={{ control: secondSwiper }}`

I'd also like to know if it's possible to link 4 swipers with each other, using the same method.
Thanks.

**Comment by TakhyunKim** [comment]
I had the same issue.
The error that occurs seems to have been fixed in the 9.2.0 version.
Here's a link to the issue, if you'd like to check it out.

- [Controller module + loop issue](https://github.com/nolimits4web/swiper/issues/6491)
- [Controller destroyed issue](https://github.com/nolimits4web/swiper/issues/6552)

If you test by changing from swiper 9.1.1 to 9.2.0 on codesandbox, you should see the issue resolved.
_(If you press the refresh button in the swiper library field, it will change to 9.2.0 version)_

- [swiper bug testing codesandbox](https://codesandbox.io/s/swiper-testing-bcm44v?file=/src/App.tsx)
<img width="310" alt="스크린샷 2023-04-02 오후 10 02 57" src="https://user-images.githubusercontent.com/64253365/229354559-286db9f7-5d64-49cf-9641-63ec33529700.png">
